README.md

```diff
@@ -1,78 +1,12 @@
-#  Reference Browser
+#  Daisy
 
-[![Task Status](https://github.taskcluster.net/v1/repository/mozilla-mobile/reference-browser/master/badge.svg)](https://github.taskcluster.net/v1/repository/mozilla-mobile/reference-browser/master/latest)
+Daisy is the new Android browser from [Cliqz](https://cliqz.com/). Daisy is a makeshift name and _this is not a production ready product!_
```

Comment:
Replace with:

Daisy is the new Android browser from [Cliqz](https://cliqz.com/). Daisy is currently under development and _is not yet a production ready product!_